dapr/proto/runtime/v1/dapr.proto

```diff
@@ -1295,6 +1295,9 @@ message ConversationAlpha1Request {
 
   // The metadata passing to conversation components.
   map<string, string> metadata = 5;
+
+  // Scrub PII data that comes back from the LLM
+  optional bool piiScrubOutput = 6;
```

Comment:
Nit: 1. Is `Output` seems superfluous here (?), 2. make first word the verb, 3. Capitalize acronym  

```suggestion
  optional bool scrubPII = 6;
```

Reply:
Agree, field name changed